### ISSUE TITLE
Change event doesn't fire on autofill

### ISSUE THREAD

**Issue by dhritzkiv** [AmpersandJS/ampersand-input-view#2]
When filling out an input field, sometimes the browser presents me with past values. On accepting these values, the input does not have events fire, leaving the last value that was there.

Browser: Safari 7.0.4

Screenshot of value drop down:
![screen shot 2014-07-10 at 6 16 56 pm](https://cloud.githubusercontent.com/assets/1349865/3546161/12b2bc42-0880-11e4-85b6-d2bf78f72040.png)

Screenshot of returned value via console (no event fired, either);
![screen shot 2014-07-10 at 6 17 30 pm](https://cloud.githubusercontent.com/assets/1349865/3546169/216125da-0880-11e4-98b1-82b12d01992b.png)

_Note: the field with the label `Address 2` is bound to the property `street2`_

**Comment by cdaringe** [comment]
@kamilogorek @HenrikJoreteg, gar and I discussed handling cases where DOM events aren't fired when expected-or-desired [here](https://github.com/AmpersandJS/ampersand-input-view/pull/43), but agreed to close the PR after discussion.  unfortunately, it seems like workaround solutions out there are equally poor.  this alone makes me want to re-open that PR, and denote why we needed to do it until Safari get's its butt in gear.
